pkg/fixtures/dackbox.go

```diff
@@ -0,0 +1,1128 @@
+package fixtures
+
+import (
+	"github.com/gogo/protobuf/types"
+	"github.com/stackrox/rox/generated/storage"
+)
+
+///////////////////////////////
+// Dackbox synthetic dataset //
+///////////////////////////////
+
+/////////////////////
+// Image with CVEs //
+
+// Data relationships
+//
+//     Namespace (scope information holder)
+//         ^ 1
+//         |
+//         v *
+//     Deployment
+//         ^ *
+//         |
+//         v *
+//       Image <--------------+ *
+//         ^ *                |
+//         |                  |
+// ImageComponentEdge         |
+//         |                  |
+//         v *                |
+//   ImageComponent      ImageCVEEdge
+//         ^ *                |
+//         |                  |
+//  ComponentCVEEdge          |
+//         |                  |
+//         v *                |
+//        CVE <---------------+ *
```

Comment:
Maybe we should have a file with fill schema diagram generated from source? What do you think about it?